cypress.config.ts

```diff
@@ -0,0 +1,21 @@
+/**
+ * @license Copyright (c) 2003-2023, CKSource Holding sp. z o.o. All rights reserved.
+ * For licensing, see LICENSE.md.
+ */
+
+import { defineConfig } from 'cypress';
+
+export default defineConfig( {
+	e2e: {
+		'baseUrl': 'http://localhost:4200',
+		supportFile: false
```

Comment:
Do we need video captured during E2E tests? If not, then it can be disabled:

```suggestion
		supportFile: false,
		video: false
```